src/style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --cardMainColor: #060b35;
+  --cardSecondarycolor: #616070;
+  --buttonColor: #00acdc;
+}
+
+.page {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  font-family: "Roboto", helvetica, sans-serif;
+  border: 1px solid #f3f3f3 ;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  object-position: center;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--cardMainColor);
+  padding: 0 0 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--cardSecondarycolor);
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 16px 0 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.reviews {
```

Comment:
<img width="697" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/27935c1d-4fc9-43e6-93b9-fb08ff9abad6">
no need to create bem block here

Reply:
also, check this checklist point
<img width="505" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/c87d4d48-d937-4679-b47e-3ce6df6bf6cd">